<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgQuickAdvanceAction</class>
 <widget class="QDialog" name="DlgQuickAdvanceAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temporary tax</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Receipt</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Item code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EQLineEdit" name="leItemCode">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Item name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EQLineEdit" name="leItemName">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Adg</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EQLineEdit" name="leAdg">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Dept</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EQLineEdit" name="leDep">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Qty</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leQty">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="lePrice">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leTotal">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnPlus">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="EQTableWidget" name="tbl">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Adg</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dept</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qty</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Grand total</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leGrandTotal">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Cash</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leGrandCash">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leGrandCard">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Cash</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leaCash">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Card</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDoubleEdit" name="leaCard">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ptComment">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="placeholderText">
      <string>Remarks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Tax number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EQLineEdit" name="leTaxNumber">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>340</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>340</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>EDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leItemCode</tabstop>
  <tabstop>leDep</tabstop>
  <tabstop>leQty</tabstop>
  <tabstop>lePrice</tabstop>
  <tabstop>leTotal</tabstop>
  <tabstop>btnPlus</tabstop>
  <tabstop>leGrandTotal</tabstop>
  <tabstop>leGrandCash</tabstop>
  <tabstop>leGrandCard</tabstop>
  <tabstop>leTaxNumber</tabstop>
  <tabstop>leaCash</tabstop>
  <tabstop>leaCard</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
